### DISCUSSION TITLE
Погоди, нужно спрятать патроны!

### DISCUSSION THREAD

**Discussion by ExitMyCover** [octothorp-team/public-dobrograd#621] [Предложения]
So....
Идея до ужаса проста, однако в какой-то степени сможет добавить новых идей во взаимодействие. 
Суть ее заключается в том, чтобы игрок, держа любой вид оружия, (искл холодное) мог изъять магазин из оружия в любой момент. Сделать это можно будет перейдя в c-меню и перейти во вкладку "оружие".
Так, при изъятии магазина в инвентарь будут переведены все патроны с данного оружия. Если места в инвентаре нет, то по моей задумке патроны будут либо выкидываться, либо в чат будет появляться сообщение, по типу: "Прятать патроны некуда!", ну или просто "Здесь недостаточно места.".
Как же поступать с абузом коповских патрон? Опять же, по МОЕЙ идеи при попытке переноса, условно, патрон легкого калибра с гражданского на служебное оружие, при попытке засунуть туда патроны абузом, тебе в чат будет приходить сообщение по типу: "Этот тип патрон не подходит для твоего оружия!" 
Ну и понятное дело, что с каждого изъятого оружия будет "дропаться" именно тот калибр, которым он "питается"
=)

**Comment by JamesTytor** [comment]
Идея хорошая но,не хотелось что бы патроны выкидывались,а вот что ты предложил что тебе писали в чат если у тебя нет мест в инвентаре,вот это надо использовать.Почему я так говорю?Просто может с тобой стоит какой то человек и у тебя выпали патроны, потому что нет мест в инвентаре, и может он украдет твои патроны,поэтому ради безопасности лучше бы тебе писали в чат.А так идея повторяюсь хорошая ставлю "За".
